digikeyscraper/core.py

```diff
@@ -23,10 +23,12 @@
 logger = logging.getLogger(__name__)
 
 
-# TODO: add try and except to account for possible requests errors
-def resolve_url(orig_url, session):
-    r = session.head(orig_url, allow_redirects=True)
-    return r.url
+def resolve_url(orig_url, session, timeout):
```

Comment:
I'd recommend making timeout a named (and thus, optional) parameter.
```suggestion
def resolve_url(orig_url, session, timeout=5):
```